zeebe/protocol/src/main/java/io/camunda/zeebe/protocol/record/value/JobListenerEventType.java

```diff
@@ -33,5 +33,39 @@ public enum JobListenerEventType {
    * the listener should be triggered at the end of an execution, such as the completion of a
    * process instance, sub-process or element (task, event, gateway).
    */
-  END
+  END,
+
+  /**
+   * Represents the `create` event for a task listener. This event type is used to indicate that the
+   * listener should be triggered when a user task is created. This occurs when the task becomes
+   * available in the workflow for the first time.
+   */
+  CREATE,
```

Comment:
❓ Why did we choose `CREATE` over `CREATING`, `CREATED`, or `CREATION`? 

To me, `create` is a command or a request. It does not describe an event. Events are things that have happened. They are typically written in the past-tense. As this feature allows blocking the operation, you could argue that the event that has happened is `CREATING` as the listener still has the chance to block the task from actually being `CREATED`. 

An alternative naming could be to use the concept that it refers to: `CREATION`. This aligns with `START` and `END` event types.

I want us to carefully consider the naming here, as this is user facing.

Reply:
Thanks for raising this point, Nico!

When I initially chose the names for `JobListenerEventType` values and [ZeebeTaskListenerEventType](https://github.com/camunda/camunda/blob/main/zeebe/bpmn-model/src/main/java/io/camunda/zeebe/model/bpmn/instance/zeebe/ZeebeTaskListenerEventType.java), I followed the same naming conventions for task listener events that we have in Camunda 7 (as seen [here](https://docs.camunda.org/manual/7.21/user-guide/process-engine/delegation-code/#task-listener-event-lifecycle)).
In Camunda 7, we use terms like `create`, `update`, `assignment`, `complete`, etc., which are already familiar to users.

However, since we're building Camunda 8, we have the opportunity to establish more consistent and clearer naming for these event types. I agree with your suggestion that naming them in a way that reflects the event's lifecycle (e.g., `CREATION ` or `COMPLETION`) could improve clarity.

We should also ensure that the naming aligns with the values in [ZeebeTaskListenerEventType](https://github.com/camunda/camunda/blob/main/zeebe/bpmn-model/src/main/java/io/camunda/zeebe/model/bpmn/instance/zeebe/ZeebeTaskListenerEventType.java), so that it consistent across the modules.

Happy to discuss this further and refine the naming to make it more intuitive for users!

I’ll raise this topic for discussion in the **#usertask-listeners** channel ✍️.

Reply:
I've started a discussion [here](https://camunda.slack.com/archives/C0622NWC9LK/p1728996247956779). Let's take the decision along here